nixos/roles/mysql.nix

```diff
@@ -462,5 +463,98 @@ in
       }
     ];
   }
+
+  (lib.mkIf (!builtins.isNull lokiServer) {
+    systemd.services.alloy = lib.mkIf config.services.alloy.enable {
+      reloadTriggers = [config.environment.etc."alloy/mysql_slowlog.alloy".source];
+      serviceConfig.SupplementaryGroups = ["service"];
+    };
+
+    environment.etc."alloy/mysql_slowlog.alloy".text = ''
+        local.file_match "mysql_slowlogs" {
+            path_targets = [{
+                __path__       = "/var/log/mysql/mysql.slow",
+                job_name       = "mysql/slowlogs",
+            }]
+        }
+
+        loki.source.file "mysql_slowlogs" {
+            targets               = local.file_match.mysql_slowlogs.targets
+            forward_to            = [loki.process.mysql_slowlogs.receiver]
+        }
+
+        loki.process "mysql_slowlogs" {
+            forward_to = [loki.write.fcio_rg_loki.receiver]
+
+            stage.multiline {
+                firstline     = "^# Time: (?P<time>.+?)"
```

Comment:
nit: there's multiple "weird" whitespace choices here ;)

Reply:
ah fair, I just copied the bulk of this from what alloy generated for the previous prometheus config and removed some fields that were not required or straight up caused issues when attempting to run it